tests/Unit/Evolution/Systems/GeneralizedHarmonic/UpwindPenaltyCorrection.py

```diff
@@ -0,0 +1,78 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+
+def pi_upwind_penalty_correction(constraint_gamma2, v_spacetime_metric_int,
+                                 v_zero_int, v_plus_int, v_minus_int,
+                                 char_speeds_int, unit_normal_int,
+                                 v_spacetime_metric_ext, v_zero_ext,
+                                 v_plus_ext, v_minus_ext, char_speeds_ext,
+                                 unit_normal_ext):
+    result = v_spacetime_metric_int * 0.
+
+    # Add v^+ terms
+    if char_speeds_ext[2] > 0.:
```

Comment:
Nitpick, but the help text says >=0, whereas here you use > 0. Since you're multiplying, the same thing happens for the case of 0, but maybe it would be nice to be consistent?